<ui version="4.0" >
 <class>VocableQuiz</class>
 <widget class="QWidget" name="VocableQuiz" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quiz</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="quizDialogStackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="quizPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="nativeLabel" >
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>21</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="buttonsStack" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="checkPage" >
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="1" >
            <widget class="QPushButton" name="checkButton" >
             <property name="text" >
              <string>&amp;check</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>191</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2" >
            <widget class="QLineEdit" name="foreignLineEdit" >
             <property name="font" >
              <font>
               <family>Sans Serif</family>
               <pointsize>21</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="nextPage" >
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="2" column="3" >
            <widget class="QPushButton" name="editButton" >
             <property name="text" >
              <string>&amp;edit</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3" >
            <widget class="QLabel" name="resultLabel" >
             <property name="font" >
              <font>
               <family>Sans Serif</family>
               <pointsize>21</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QPushButton" name="correctButton" >
             <property name="toolTip" >
              <string>mark correct and move to next vocable</string>
             </property>
             <property name="text" >
              <string>&amp;correct</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3" >
            <widget class="QPushButton" name="nextButton" >
             <property name="text" >
              <string>&amp;next</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="resultTextLabel" >
             <property name="text" >
              <string>resultLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="helpLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="waitPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Please wait until there are expired vocables.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="waitLabel" >
         <property name="font" >
          <font>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>421</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>foreignLineEdit</tabstop>
  <tabstop>checkButton</tabstop>
  <tabstop>correctButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
